ynr/apps/sopn_parsing/helpers/parse_tables.py

```diff
@@ -125,6 +125,7 @@ def clean_last_names(names):
 
 def clean_description(description):
     description = str(description)
+    description = description.replace(" & ", " and ")
```

Comment:
This does work in the case of this SOPN/ballot. My concern would be that it could break others, when the party name/description we are looking for actually should have the `&` in it rather than `and`. As there are some parties/descriptions stored in the DB with the `&`, but I dont know how common they appear.
```
In [12]: Party.objects.filter(name__icontains="&").count()
Out[12]: 26

In [13]: PartyDescription.objects.filter(description__icontains="&").count()
Out[13]: 47
```

So it would be interesting to see how this affects the numbers overall when running `make test-sopns` for 2021 after #1706 has been merged